yoti/yoti.pages.inc

```diff
@@ -8,19 +8,9 @@
 require_once __DIR__ . '/YotiHelper.php';
 
 /**
- * Authenticate Yoti user or unlink if user is already linked to Yoti user.
+ * Authenticate Yoti user and link to current user account.
```

Comment:
Single Responsibility Principle - could we split this up into "authenticate" and "link"?

Reply:
As this is the page callback primarily calling `$helper->link()`, I'm not sure we can split this up much more. I can reword this along the lines of:
_"Page callback to handle linking of Yoti user to current user_"